AdyenDropIn/Modules/ComponentContainer/ComponentContainerRouter.swift

```diff
@@ -48,9 +49,18 @@ internal class ComponentContainerRouter: Router, ComponentContainerRouting {
     // MARK: - ComponentContainerRouting
     
     internal func present(component: any PresentableComponent) {
-        rootViewController.present(component.viewController, animated: true)
+        let componentViewController = component.viewController
+        rootViewController.navigationController?.pushViewController(componentViewController, animated: true)
     }
-    
+
+    internal func present(actionComponent: any PresentableComponent, onCancel: (() -> Void)?) {
+        let actionViewController = ActionPresentationHelper.viewController(
+            for: actionComponent,
+            onCancel: onCancel
+        )
+        rootViewController.present(actionViewController, animated: true)
```

Comment:
This presentation example is fine, as it calls a generic `present` method that could be easily refactored in something like `presenter.present(actionViewController, animated: true)`, keeping the same API while easily changing the way the screen is presented - push, modal, etc